schema/event_pointset.tests/writeback.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": 1,
+  "timestamp": "2018-08-26T21:39:29.364Z",
+  "etag": "2a8b718",
```

Comment:
I don't think etag is strictly necessary in the telemetry message. If the cloud wants to figure out what state the device was in when a given telemetry message was sent, the cloud can compare timestamps. I won't block on this, but WDYT?